files/generate_vars.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+import configparser
+import subprocess
+import shlex
+
+
+class GenerateMappingFile():
+    prefix = "[Generate Mapping File] "
+
+    def run(self):
+        print("\n%sStart Generate Varialbe Mapping File "
+              "For oVirt Ansible Disaster Recovery\n" % self.prefix)
+        site, username, password, ca, output_file = self._init_vars()
+        print("%ssite address: %s \n"
+              "%susername: %s \n"
+              "%spassword: %s \n"
+              "%sca file location: %s \n"
+              "%soutput file location: %s \n"
+              % (self.prefix,
+                 site,
+                 self.prefix,
+                 username,
+                 self.prefix,
+                 password,
+                 self.prefix,
+                 ca,
+                 self.prefix,
+                 output_file))
+
+        ansible_play = "../examples/dr_generate_var.yml"
+        dr_tag = "generate_mapping"
+        external_vars = "site={} username={} password={} ca={} var_file={}" \
+                        .format(site, username, password, ca, output_file)
+        command = "ansible-playbook {} -t {} -e '{}'" \
+                  .format(ansible_play, dr_tag, external_vars)
+        process = subprocess.Popen('/bin/bash', stdin=subprocess.PIPE,
```

Comment:
Perhaps better to run the command directly and not bash.

Reply:
done